src/cli/domain/local.js

```diff
@@ -57,7 +57,7 @@ module.exports = function() {
             name: componentName
           });
 
-        fs.outputJsonSync(componentPath, component);
+        fs.outputJsonSync(componentPath, component, { spaces: '  ' });
```

Comment:
as per https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputJson-sync.md#outputjsonsyncfile-object-options shall we use `{ spaces: 2 }` instead?

this will add also `\n`s?